homeassistant/components/mpd/media_player.py

```diff
@@ -163,6 +165,7 @@ async def async_update(self):
         try:
             if not self._is_connected:
                 await self._connect()
+                self._commands = list(await self._client.commands())
```

Comment:
What is the original return value before the list copy?

Reply:
A generator.